### PR TITLE
feat: Added Filter by feature page and it's css

### PR DESCRIPTION
**Pull request by zabihshaik**
Added Filter by Feature Page and imported it's css file in styles.css.

Filter by feature Page looks like:

Filter Page

### CODE REVIEW

File: styles/filter.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+.inputs {
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.radio-unchecked {
+    margin: 0.4rem;
```
Comment (Sharmaryan) [inline styles/filter.css]: values less than 1rem can be written in  px (upto 10px)

**Review by Sharmaryan — APPROVED** [review]
looks good to merge

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

File: styles/filter.css (hunk 1 above)
Comment (parvez-rashid) [inline styles/filter.css]: You can use px here because generally scaling a small value is not desired.

File: styles/filter.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+.slider {
+    width: 100%;
+    height: 2rem;
+    border-radius: 5px;
+    opacity: 0.9;
+    transition: opacity .2s;
+}
+
+.slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
```
Comment (parvez-rashid) [inline styles/filter.css:55]: You don't have to use browser specific (`-webkit-`) styles, that is handled automatically.

File: styles/filter.css (hunk 1 above)
Comment (zabihshaik) [reply styles/filter.css]: Yes. Thanks for suggestion.

**Review by parvez-rashid — APPROVED** [review]
UI Review: The UI looks great. I've commented below for the code, please have a look. Rest looks good, can be merged.

File: styles/filter.css (hunk 1 above)
Comment (zabihshaik) [reply styles/filter.css]: Yes. Modified it. Thanks.

File: styles/filter.css (hunk 2 above)
Comment (zabihshaik) [reply styles/filter.css:55]: Ok. Will do it.

**Pull request merged**